src/checkers/inference/VariableAnnotator.java

```diff
@@ -825,7 +825,7 @@ private void handleClassDeclaration(AnnotatedDeclaredType classType, ClassTree c
      */
     private AnnotationLocation createImpliedExtendsLocation(ClassTree classTree) {
         // TODO: THIS CAN BE CREATED ONCE THIS IS FIXED: https://github.com/typetools/annotation-tools/issues/100
```

Comment:
The references issue has been fixed.
I assume we currently don't need annotations on extends clauses.

Reply:
Should we simply remove the TODO and the logger warning OR fix the caller:
https://github.com/opprop/checker-framework-inference/blob/4101d926b6f59bf945a3ee07574ce10770fb9bd8/src/checkers/inference/VariableAnnotator.java#L772
Specifically method `VariableAnnotator#handleClassDeclaration`?

Reply:
Keep as warning.